Program.cs

```diff
@@ -0,0 +1,238 @@
+﻿using RefrigeratorExercise.Etities;
+using System;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+
+public class Program
+{
+    private static Refrigerator refrigerator;
+
+    public IEnumerable<Shelf> Shelves { get; private set; }
+
+    public static void Main(string[] args)
+    {
+        Program program = new Program();
+        program.run();
+    }
+    public void run()
+    {
+        refrigerator = new Refrigerator();
+
+        Shelf shelf1 = new Shelf( 1, 7);
+        Shelf shelf2 = new Shelf( 2, 10);
+        Shelf shelf3 = new Shelf( 3, 18);
+
+        refrigerator.Shelves = new List<Shelf> { shelf1, shelf2, shelf3};
+
+        Item item1 = new Item("milk", 2.0, "2023-11-16", TypeOfFood.Drinking, Kosher.Dairy);
+        Item item2 = new Item("chocolate",5.0, "2023-10-02", TypeOfFood.Food, Kosher.Dairy);
+        Item item3 = new Item("cheeze", 3, "2023-11-11",TypeOfFood.Food, Kosher.Dairy);
+        Item item4 = new Item("yugort", 2, "2023-10-24",TypeOfFood.Food, Kosher.Dairy);
+        Item item5 = new Item("soup", 4, "2023-11-21",TypeOfFood.Food, Kosher.Meats);
+        refrigerator.AddItemToRefrigerator( item1);
+        refrigerator.AddItemToRefrigerator( item2);
+        refrigerator.AddItemToRefrigerator( item3);
+        refrigerator.AddItemToRefrigerator( item4);
+        refrigerator.AddItemToRefrigerator( item5);
+
+        bool running = true;
+
+        while (running)
+        { 
+            Console.WriteLine("Press 1: Print all items in the refrigerator");
+            Console.WriteLine("Press 2: Print remaining space in the fridge");
+            Console.WriteLine("Press 3: Add an item to the fridge");
+            Console.WriteLine("Press 4: Remove an item from the refrigerator");
+            Console.WriteLine("Press 5: Clean the refrigerator and print removed items");
+            Console.WriteLine("Press 6: Request a product from the fridge");
+            Console.WriteLine("Press 7: Print all products sorted by expiration date");
+            Console.WriteLine("Press 8: Print shelves arranged by free space");
+            Console.WriteLine("Press 9: Print refrigerators arranged by free space");
+            Console.WriteLine("Press 10: Prepare the refrigerator for shopping");
+            Console.WriteLine("Press 100: Shutdown the system");
+
+            Console.WriteLine("Enter your choice:");
+
+            string choice = Console.ReadLine();
+
+            switch (choice)
```

Comment:
all this code is not wrapped in exception handling, any small mistake will make your program crash.